trillian/docs/ManualDeployment.md

```diff
@@ -0,0 +1,460 @@
+# CT Log Deployment (Manual)
+
+This document describes the individual steps and components involved in the
+deployment of a Trillian-based CT Log.  These steps will gradually build up a
+system as shown in the diagram below.
+
+<img src="images/DeploymentFull.png" width="650">
```

Comment:
Strictly, the LogServer talks with etcd too if using the `etcd` quota mechanism.